code/modules/mob/living/carbon/human/species/station/traits_vr/neutral.dm

```diff
@@ -81,4 +81,10 @@
 	name = "Brutal Predation"
 	desc = "Allows you to tear off limbs & tear out internal organs."
 	cost = 0 //I would make this cost a point, since it has some in game value, but there are easier, less damaging ways to perform the same functions.
-	var_changes = list("hard_vore_enabled" = 1) //The verb is given in human.dm
\ No newline at end of file
+	var_changes = list("hard_vore_enabled" = 1) //The verb is given in human.dm
+
+/datum/trait/trashcan
+	name = "Trash Can"
+	desc = "Someone thought it'd be hilarious to eat the snack wrappings too instead of having to look for a bin or just littering like an animal."
```

Comment:
Description doesn't properly describe what the trait does, nor does it align with existing trait descriptions.